<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>specialOptionDialog</class>
 <widget class="QDialog" name="specialOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Special Options Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>308</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="discardButton">
       <property name="text">
        <string>discard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>253</red>
           <green>252</green>
           <blue>250</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>121</red>
           <green>116</green>
           <blue>108</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>253</red>
           <green>252</green>
           <blue>250</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>121</red>
           <green>116</green>
           <blue>108</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>121</red>
           <green>116</green>
           <blue>108</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>121</red>
           <green>116</green>
           <blue>108</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QTableWidget" name="valueTable">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>560</number>
        </attribute>
        <column/>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="sourceLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="valueLineEdit">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>clear</string>
          </property>
          <property name="icon">
           <iconset resource="../src/icons.qrc">
            <normaloff>:/icon/icons/skull.png</normaloff>:/icon/icons/skull.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>328</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="delButton">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>del</string>
          </property>
          <property name="icon">
           <iconset resource="../src/icons.qrc">
            <normaloff>:/icon/icons/del.png</normaloff>:/icon/icons/del.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>add</string>
          </property>
          <property name="icon">
           <iconset resource="../src/icons.qrc">
            <normaloff>:/icon/icons/add.png</normaloff>:/icon/icons/add.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="headerLabel">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>Option</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../src/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>discardButton</sender>
   <signal>clicked()</signal>
   <receiver>specialOptionDialog</receiver>
   <slot>slot_discardButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submitButton</sender>
   <signal>clicked()</signal>
   <receiver>specialOptionDialog</receiver>
   <slot>slot_submitButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueTable</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>specialOptionDialog</receiver>
   <slot>slot_tableClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>specialOptionDialog</receiver>
   <slot>slot_clearButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>specialOptionDialog</receiver>
   <slot>slot_addButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delButton</sender>
   <signal>clicked()</signal>
   <receiver>specialOptionDialog</receiver>
   <slot>slot_delButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_addButtonClicked()</slot>
  <slot>slot_delButtonClicked()</slot>
  <slot>slot_discardButtonClicked()</slot>
  <slot>slot_submitButtonClicked()</slot>
  <slot>slot_tableClicked()</slot>
  <slot>slot_clearButtonClicked()</slot>
 </slots>
</ui>
